tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -35,14 +39,47 @@ def test_inhibit_if_custom_module_loaded(insert_custom_kmod, convert2rhel):
 
 def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
     assert shell("modprobe -r -v bonding").returncode == 0
+    # If custom module is not loaded the conversion is not inhibited.
     with convert2rhel(
-        ("-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+        ("--no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
             env.str("RHSM_SERVER_URL"),
             env.str("RHSM_USERNAME"),
             env.str("RHSM_PASSWORD"),
             env.str("RHSM_POOL"),
         )
     ) as c2r:
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+        # On OracleLinux there is one question less than on Centos
+        if platform.platform().find("oracle-8") < 0:
```

Comment:
`if "oracle-8" in platform.platform()` seems more readable for me

Reply:
Thanks, this is a good point. I changed it.

I also changed the same occurrences in other tests so it's consistent and more readable also there. I made a separate commit for this change (https://github.com/oamg/convert2rhel/pull/344/commits/98c8da60beba63308e75b0f7230c34a73dbfbdee).